src/api.ts

```diff
@@ -311,13 +317,15 @@ router.get(
   async (req: express.Request, res: express.Response) => {
     const infuraUrl = process.env["INFURA_URL"];
     if (!infuraUrl) {
-      throw Error("No Infura URL could be found")
+      throw Error("No Infura URL could be found");
     }
     const sampleProvider = new ethers.providers.JsonRpcProvider(infuraUrl);
     const blockNumber = await sampleProvider.getBlockNumber();
 
     if (!blockNumber) {
-      throw Error("Looks like something went wrong with the Infura connection.");
+      throw Error(
+        "Looks like something went wrong with the Infura connection."
```

Comment:
We have retry logic in other places, this strikes me as a good place to retry getting the block number a few times before we throw. It could just be a timeout

Reply:
Think I just updated the formatting here but its a good point. 

Added a retry wrapper function to account for timeouts.